blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -3,19 +3,23 @@ import Immutable from 'immutable';
 since we would return a function, */
```

Comment:
as in - having a PipelineItem parent type that pipeline record and folder record descend from?